bot/exts/evergreen/wonder_twins.py

```diff
@@ -0,0 +1,49 @@
+import random
+from pathlib import Path
+
+import yaml
+from discord.ext.commands import Bot, Cog, Context, command
+
+with open(Path.cwd() / "bot" / "resources" / "evergreen" / "wonder_twins.yaml", "r", encoding="utf-8") as f:
+    info = yaml.load(f, Loader=yaml.FullLoader)
+    WATER_TYPES, OBJECTS, ADJECTIVES = info["water_types"], info["objects"], info["adjectives"]
+
+
+class WonderTwins(Cog):
+    """Cog for a Wonder Twins inspired command."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @staticmethod
+    def append_onto(phrase: str, insert_word: str) -> str:
+        """Appends one word onto the end of another phrase, used to format the use of 'an' or 'a'."""
+        if insert_word[-1] == "s":
```

Comment:
I'd prefer here for readability:
```suggestion
        if insert_word.endswith("s"):
```